server/models/users.js

```diff
@@ -0,0 +1,49 @@
+import mongoose from 'mongoose';
+
+const sectionsSchema = mongoose.Schema({
+    sectionName : String,
+    sectionID : Number,
+    sectionHeader : String,
+    sectionContent: {
```

Comment:
This will be array of objects

Reply:
bhaiya but i am creating a separate schema for it so it is already an array of records right ?